azure-pipelines.yml

```diff
@@ -8,32 +10,32 @@ pr:
     exclude:
     - README.md
     - docs/*
-
-# Run official build every day at midnight, if code has changed since the last run.
 schedules:
 - cron: "0 0 * * *"
   displayName: Daily Official Build
   branches:
     include:
     - main
-
 variables:
 - template: /eng/common/templates-official/variables/pool-providers.yml@self
 - name: _TeamName
   value: Roslyn
+- name: TeamName
+  value: Roslyn
 resources:
   repositories:
-  - repository: 1ESPipelineTemplates
+  - repository: MicroBuildTemplate
```

Comment:
Where is this repository?

Reply:
There is a sibling project in dnceng which contains the repo.